packages/wallet/src/wallet.ts

```diff
@@ -327,4 +328,53 @@ export class Wallet {
         await this.db.saveSilentPaymentAddress(address);
         return address;
     }
+
+    async scanSilentBlock(silentBlock: SilentBlock): Promise<void> {
+        const dustLimit = 546;
+        const matchedUTXOs: Coin[] = [];
+    
+        const coinType = this.network.network.bech32 === 'bc' ? 0 : 1;
+        const spendKey = this.masterKey.derivePath(`m/352'/${coinType}'/0'/0'/0`);
+        
+        const spendPublicKey = spendKey.publicKey;
+    
+        for (const transaction of silentBlock.transactions) {
+            const outputs = transaction.outputs.filter((output) => output.value >= dustLimit);
+    
+            if (outputs.length === 0) continue;
+    
+            const outputPubKeys = outputs.map((output) => Buffer.from(output.pubKey, 'hex'));
+    
+            const scanTweak = Buffer.from(transaction.scanTweak, 'hex');
+    
+            const matches = scanOutputsWithTweak(
+                spendPublicKey,
+                scanTweak,
+                outputPubKeys
+            );
```

Comment:
We can replace this function call with the call to the function positioned in the comment earlier.